addons/message_survey_staging_30m/manifest.json

```diff
@@ -6,6 +6,8 @@
   "type": "message",
   "conditions": {
     "env": "staging",
+    "min_os_version": {"ios": "14.0", "macos": "12.4"},
```

Comment:
Just curious: was it not possible to hide/show the message via enable.js with `api.env.osVersion`? These changes look right to me, but would mean that if we ever wanted to show alternate UI in a single message depending on the osVersion we'd need to ship separate messages.

Reply:
We can - and per conversation with Naomi above, I'm going to go that route - but my initial goal was to find a solution that was a condition, as it's easier to re-use (in my opinion) and less prone to bugs (if the condition code is solid). But given that old clients dismiss addons with conditions they don't expect, that doesn't seem like a workable approach.